docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -249,6 +252,20 @@ subquestion: |
   The Plaintiff/Petitioner starts a case. The Defendant/Respondent
   is responding to a case someone else started.
 fields:
+  - note: |
+      The list of names below look like names for people, based on
+      how you use them in your document.
+    show if:
+      code: |
+        len(get_person_variables(all_fields, custom_only=True)) > 0
+  - Which variable names represent people?: people_variables
+    datatype: checkboxes
+    code: |
+      get_person_variables(all_fields, custom_only=True)
+    help: |
+      "person" variables will get things like addresses, multi-part
+      names, and proper questions handled automatically. Using them
+      will help you save time writing your interview.
```

Comment:
There's a separate issue for that: https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/issues/163

We can disable the ability to pre-check the extra names until #163 is ready but I'm hoping to get that in before the bootcamp starts.

Reply:
That's fair, I didn't realize there were separate issues. 

I'd lean towards disabling until #163 is ready, this is awesome, but only if you know what you're doing with the generated interview already.

Reply:
I'll do a quick patch for that after I merge this. I haven't sat down to tackle getting in the custom list of people names in yet--I think it will be straightforward but I want to track all of the places it touches in the code.

Basically any place that references `map_names()` I think